### DISCUSSION TITLE
Issues of Email Change for Students from 2024 Onwards

### DISCUSSION THREAD

**Discussion by relic-yuexi** [community/community#136774] [GitHub Education] [answered]
#### Background
Shenzhen University will be changing the student email system starting from the 2024 class. New students will automatically have an email account created with the format "student_ID@mails.szu.edu.cn" upon registering with the enterprise WeChat. The enterprise WeChat client will automatically log in to this email account. However, this change has resulted in students from the 2024 class onwards being unable to apply for GitHub Education student verification using the new email.

#### Request for Assistance
We hope that the GitHub Education team can provide assistance to address the following issues:

1. **Email Format Change**: The new email format is "student_ID@mails.szu.edu.cn" instead of the previous "student_ID@szu.edu.cn".
2. **Student Verification Issue**: Due to the email format change, students from the 2024 class onwards are unable to apply for GitHub Education student verification using the new email.

#### Specific Requests
1. **Update Email Format Verification**: Please update the email format verification system to support the new email format "student_ID@mails.szu.edu.cn".
2. **Temporary Solution**: Is there a temporary solution that can help students from the 2024 class onwards complete the student verification process while the system is being updated?

#### Notice Documentation
This notice is documented at the following URL: [https://www1.szu.edu.cn/nc/view.asp?id=65](https://www1.szu.edu.cn/nc/view.asp?id=65)

---

Thank you for your attention and assistance.

**Comment by samus-aran** [comment, marked as answer]
Hi @relic-yuexi  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

You should provide evidence to support if your school has additional academic domains, has changed its academic email issuing policy, or does not provide academic emails. Reach out to your institution's IT team for this. 

Hope this helps! Let us know how you get along 👍
